CtrlVAF/CtrlVAF.Core/Validators/ValidatorDispatcher.cs

```diff
@@ -27,7 +25,7 @@ public override IEnumerable<ValidationFinding> Dispatch(params ICtrlVAFCommand[]
 
             var types = GetTypes(commands);
 
-            return HandleConcreteTypes(types, commands);
+            return HandleConcreteTypes(types, commands).ToArray();
```

Comment:
Hmm, this must be a consequence of the way findings are returned from HandleConcreteTypes. Without it, the code will likely continue to execute and perhaps finish before all findings have been yield returned back.

I think this is ok for now